build.sbt

```diff
@@ -11,7 +11,7 @@ scalaVersion := "2.10.4"
 scalacOptions += "-feature"
 
 libraryDependencies ++= Seq(
-  "org.webjars" % "rjs" % "2.1.15"
+  "org.webjars" % "requirejs-node" % "2.1.17"
```

Comment:
`"org.webjars.npm" % "requirejs" % "2.2.0"` seems like the right thing to do. Keep in mind this PR is over a year old. Happy to close this PR.